packages/react-instantsearch-core/src/connectors/__tests__/connectMenu.js

```diff
@@ -416,7 +416,53 @@ describe('connectMenu', () => {
       });
     });
 
-    it('when search for facets values is activated order the item by isRefined first', () => {
+    it('if not searchable: uses a static sortBy order', () => {
+      const results = {
+        getFacetValues: jest.fn(() => []),
+        getFacetByName: () => true,
+        hits: [],
+      };
+      results.getFacetValues.mockClear();
+      results.getFacetValues.mockImplementation(() => [
```

Comment:
Why not inlining the mock implementation inside the declaration?

Reply:
I didn't write this, only edited it. I tried out using a real helper, but it was a lot of boilerplate and much harder to set up by the way